Dockerfile

```diff
@@ -1,45 +1,12 @@
-# Build image
-FROM python:3.7-slim as builder
-
-# Setup virtualenv
-ENV VIRTUAL_ENV=/opt/venv
-RUN python -m venv $VIRTUAL_ENV
-ENV PATH="$VIRTUAL_ENV/bin:$PATH"
-
-# Install build deps
+FROM python:3.9-slim
 RUN apt-get update
-RUN apt-get install -y --no-install-recommends build-essential pkg-config libpq-dev
-RUN python -m pip install --upgrade pip
-RUN pip install wheel
-
-# Install runtime deps
-COPY requirements.txt /tmp/requirements.txt
-RUN pip install -r /tmp/requirements.txt
-
-# Install c-lightning specific deps
-RUN pip install pylightning
-
-# Install LND specific deps
-RUN pip install lndgrpc
-
-# Production image
-FROM python:3.7-slim as lnbits
-
-# Run as non-root
-USER 1000:1000
-
-# Copy over virtualenv
-ENV VIRTUAL_ENV="/opt/venv"
-COPY --from=builder --chown=1000:1000 $VIRTUAL_ENV $VIRTUAL_ENV
-ENV PATH="$VIRTUAL_ENV/bin:$PATH"
-
-# Copy in app source
+RUN apt-get install -y curl
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH="/root/.local/bin:$PATH"
+COPY . .
+RUN poetry config virtualenvs.create false
+RUN poetry install --no-dev --no-root
 WORKDIR /app
-COPY --chown=1000:1000 lnbits /app/lnbits
-
-ENV LNBITS_PORT="5000"
-ENV LNBITS_HOST="0.0.0.0"
-
+COPY lnbits /app/lnbits
```

Comment:
- does this overlap with `COPY . .`?
- will `lnbits` be copied both in `~/` and `/app/lnbits`?

Reply:
:thinking: i dont know